db/initdb.sh

```
@@ -4,6 +4,16 @@ set -b
 MOXDIR=${BASE_DIR}
 DIR=${DB_DIR}
 
+test -z "$DIR" && DIR=$(cd $(dirname $0); pwd)
+test -z "$BASE_DIR" && BASE_DIR=$(cd $(dirname $DIR); pwd)
+test -z "$PYTHON_EXEC" && PYTHON_EXEC=${BASE_DIR}/python-env/bin/python
+test -z "$SUPER_USER" && SUPER_USER=postgres
+test -z "$MOX_DB" && MOX_DB=mox
+test -z "$MOX_DB_USER" && MOX_DB_USER=mox
+test -z "$MOX_DB_PASSWORD" && MOX_DB_PASSWORD=mox
+
```

Preferably the script should fail to execute rather than having fallback values, in a production setting the fallback values will not work - however its a lot better than it was before